tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -455,22 +456,80 @@ def test_action_monitoring_triggered_event_handler_does_trigger_monitor_call(  #
     # add MR to DB, with nonce being larger than in event_new_bp
     context.db.upsert_monitor_request(get_signed_monitor_request(nonce=6))
 
-    event4 = ActionMonitoringTriggeredEvent(
+    trigger_event = ActionMonitoringTriggeredEvent(
         token_network_address=DEFAULT_TOKEN_NETWORK_ADDRESS,
         channel_identifier=DEFAULT_CHANNEL_IDENTIFIER,
         non_closing_participant=DEFAULT_PARTICIPANT2,
     )
 
-    channel = context.db.get_channel(event4.token_network_address, event4.channel_identifier)
+    channel = context.db.get_channel(
+        trigger_event.token_network_address,
+        trigger_event.channel_identifier,
+    )
     assert channel
     assert channel.update_status is not None
     assert channel.closing_tx_hash is None
 
-    action_monitoring_triggered_event_handler(event4, context)
+    user_address = DEFAULT_PARTICIPANT2
+    context.user_deposit_contract.functions.effectiveBalance(user_address).call.return_value = 21
+    action_monitoring_triggered_event_handler(trigger_event, context)
 
     # check that the monitor call has been done
     assert context.monitoring_service_contract.functions.monitor.called is True
 
+def test_action_monitoring_triggered_event_handler_without_suff_balance_doesnt_trigger_monitor_call(  # noqa
+    context: Context,
+):
+    """ Tests that `monitor` is called when the ActionMonitoringTriggeredEvent is triggered
+
+    Also a test for https://github.com/raiden-network/raiden-services/issues/29 , as the MR
+    is sent after the channel has been closed.
+    """
+    context = setup_state_with_closed_channel(context)
+
+    event_new_bp = ReceiveMonitoringNewBalanceProofEvent(
```

Comment:
This thing can be removed, there is another test for this.

Reply:
OK I removed it